drivers/hwtrig/hwtrig_stm32_timer.c

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (c) 2016 Linaro Limited.
+ * Copyright (c) 2020 Teslabs Engineering S.L.
```

Comment:
I wonder why I'm listed in the copyright here

Reply:
I copied all the code from the pwm driver and added the magic line `LL_TIM_SetTriggerOutput`, so I kept the copyrights